content/source/docs/github-actions/getting-started/index.html.md

```diff
@@ -31,54 +31,39 @@ Terraform's GitHub Actions on new and updated pull requests.
 
 1. Replace the default workflow with the following:
 
-    ```hcl
-    workflow "Terraform" {
-      resolves = "terraform-plan"
-      on = "pull_request"
-    }
-
-    action "filter-to-pr-open-synced" {
-      uses = "actions/bin/filter@master"
-      args = "action 'opened|synchronize'"
-    }
-
-    action "terraform-fmt" {
-      uses = "hashicorp/terraform-github-actions/fmt@v<latest version>"
-      needs = "filter-to-pr-open-synced"
-      secrets = ["GITHUB_TOKEN"]
-      env = {
-        TF_ACTION_WORKING_DIR = "."
-      }
-    }
-
-    action "terraform-init" {
-      uses = "hashicorp/terraform-github-actions/init@v<latest version>"
-      needs = "terraform-fmt"
-      secrets = ["GITHUB_TOKEN"]
-      env = {
-        TF_ACTION_WORKING_DIR = "."
-      }
-    }
-
-    action "terraform-validate" {
-      uses = "hashicorp/terraform-github-actions/validate@v<latest version>"
-      needs = "terraform-init"
-      secrets = ["GITHUB_TOKEN"]
-      env = {
-        TF_ACTION_WORKING_DIR = "."
-      }
-    }
-
-    action "terraform-plan" {
-      uses = "hashicorp/terraform-github-actions/plan@v<latest version>"
-      needs = "terraform-validate"
-      secrets = ["GITHUB_TOKEN"]
-      env = {
-        TF_ACTION_WORKING_DIR = "."
-        # If you're using Terraform workspaces, set this to the workspace name.
-        TF_ACTION_WORKSPACE = "default"
-      }
-    }
+    ```yaml
+    on: pull_request
+    name: Terraform
+    jobs:
+      filter-to-pr-open-synced:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@master
+        - name: filter-to-pr-open-synced
+          uses: actions/bin/filter@master
+          with:
+            args: action 'opened|synchronize'
```

Comment:
This step is outdated in the YAML syntax.  We've added event filtering directly to the YAML.  You can achieve this by movig the types up into the `on` trigger above.  In this case:

```yaml
on:
  pull_request:
    types: [ opened, synchronize ]
```

However, the default triggers for `on: pull_request` are actually [opened, synchronized, reopened](https://help.github.com/en/articles/events-that-trigger-workflows) which seem like reasonable defaults here, too, so you might just remove this step and not update the `on: pull_request` line at all.

Reply:
Additionally with the latest release of 0.4.2, you no longer have to run these checks exclusively on PRs. You can just run it on a normal push